sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyDetectionConfigurationLiveTests.cs

```diff
@@ -578,7 +577,7 @@ public async Task UpdateDetectionConfigurationWithMinimumSetupAndNewInstance()
         }
 
         [RecordedTest]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/21177")]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/21509")]
```

Comment:
Maybe a CodeGen bug (https://github.com/Azure/azure-sdk-for-net/issues/21509), but I couldn't overwrite the autogenerated serialization method. We need to do so to make Update methods work properly. I'll investigate if it's on our side.